### DISCUSSION TITLE
Not updating @babel/core

### DISCUSSION THREAD

**Discussion by eps1lon** [renovatebot/renovate#10968] [Request Help] [answered]
We currently have `@babel/core@7.14.5` installed while `7.14.8` is available. However, while updates have been posted for other dependencies, `@babel/core` was not.,

Repository: https://github.com/mui-org/material-ui/ ([permalink](https://github.com/mui-org/material-ui/tree/4b128c53b895b863e5fd7c4069b53c1598203421))

https://github.com/mui-org/material-ui/blob/4b128c53b895b863e5fd7c4069b53c1598203421/.github/renovate.json:
<details>
<summary>Renovate config</summary>

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "automerge": false,
  "commitMessageAction": "Bump",
  "commitMessageExtra": "to {{newValue}}",
  "commitMessageTopic": "{{depName}}",
  "dependencyDashboard": true,
  "ignoreDeps": [],
  "labels": ["dependencies"],
  "packageRules": [
    {
      "groupName": "babel",
      "matchPackagePatterns": "@babel/*"
    },
    {
      "groupName": "date-io",
      "matchPackagePatterns": "@date-io/*"
    },
    {
      "groupName": "Emotion",
      "matchPackagePatterns": "@emotion/*"
    },
    {
      "groupName": "core-js",
      "matchPackageNames": ["core-js"],
      "allowedVersions": "< 2.0.0"
    },
    {
      "groupName": "JSS",
      "matchPackageNames": [
        "jss",
        "jss-plugin-camel-case",
        "jss-plugin-default-unit",
        "jss-plugin-global",
        "jss-plugin-nested",
        "jss-plugin-props-sort",
        "jss-plugin-props-sort",
        "jss-plugin-rule-value-function",
        "jss-plugin-vendor-prefixer",
        "react-jss"
      ]
    },
    {
      "groupName": "raw-loader",
      "matchPackageNames": ["raw-loader"],
      "allowedVersions": "< 2.0.0"
    },
    {
      "groupName": "react-spring",
      "matchPackageNames": ["react-spring"],
      "allowedVersions": "< 9.0.0"
    },
    {
      "groupName": "React",
      "matchPackageNames": ["react", "react-dom", "react-is", "react-test-renderer"]
    },
    {
      "groupName": "tslint",
      "matchPackageNames": ["tslint"],
      "allowedVersions": "<= 5.14.0"
    },
    {
      "groupName": "typescript-eslint",
      "matchPackagePatterns": "@typescript-eslint/*"
    },
    {
      "groupName": "bundling fixtures",
      "matchPaths": ["test/bundling/fixtures/**/package.json"],
      "schedule": "every 6 months on the first day of the month"
    },
    {
      "groupName": "examples",
      "matchPaths": ["examples/**/package.json"],
      "minor": {
        "enabled": false
      },
      "patch": {
        "enabled": false
      }
    }
  ],
  "postUpdateOptions": ["yarnDedupeHighest"],
  "prConcurrentLimit": 30,
  "prHourlyLimit": 0,
  "schedule": "on sunday before 6:00am",
  "timezone": "UTC"
}
```

</details>

Could not find any error/warning related to `@babel/core` in the [latest log](https://app.renovatebot.com/dashboard#github/mui-org/material-ui/411153435).

**Comment by rarkins** [comment, marked as answer]
Please check the docs for `rangeStrategy`. The default value is `replace` but it sounds like you're perhaps expecting `update-lockfile`.

**Reply by eps1lon** [reply]
> Please check the docs for `rangeStrategy`. The default value is `replace` but it sounds like you're perhaps expecting `update-lockfile`.

Makes sense, thanks.

**Reply by eps1lon** [reply]
Though on a different note: Might be helpful to include in the logs how `rangeStrategy` affects the renovate behavior. Right now I'd have to read the complete config docs to understand why renovate behaves that way.

**Reply by rarkins** [reply]
Where would you suggest would be most helpful to add such a note?

**Reply by eps1lon** [reply]
> Where would you suggest would be most helpful to add such a note?

Sorry, had a typo:

>  Might be helpful to include in the logs how rangeStrategy affects the renovate behavior.
